.github/workflows/test.yml

```diff
@@ -29,12 +30,9 @@ jobs:
     - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
       with:
         node-version: '20.x'
-    - run: npm install
-    - run: npm run build
-    - run: npm test
-    - run: npm prune --production
-    - run: |
-        export BOT_AUTH_TOKEN=secret
-        for i in dist/functions/*.js; do 
-          node $i;
-        done
+    - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+    - run: pnpm install
+    - run: pnpm run build
+    - run: pnpm test
+    - run: pnpm run bundle
```

Comment:
Why do we have to bundle now when we didn't before?

Reply:
This was also an oversight in my last PR; we used to in this check test the compiled code, but that doesn't really matter, we want to verify that the bundle we're going to send to Azure even runs, just as an extra safety check.